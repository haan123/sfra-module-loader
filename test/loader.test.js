import webpack from './helpers/compiler';

describe('Loader', () => {
  test('Defaults', async () => {
    const config = {
      loader: {
        test: /(js)/,
        options: {},
      },
    };

    const stats = await webpack('fixture.js', config);
    const [module] = stats.toJson().modules;
    const { source } = module;

    expect(source).toMatchSnapshot();
  });
});
