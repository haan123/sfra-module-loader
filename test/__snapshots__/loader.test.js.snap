// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Defaults 1`] = `
"import core from '*/core';
const pluggin = require('pluggin:pluggin'); // direct request
const dump = require('dump');
const dump2 = require('./dump2');

export default () => {
  core();
  pluggin();
};
"
`;
